Source/WebGPU/WebGPU/Buffer.mm

```diff
@@ -120,6 +120,7 @@ static MTLStorageMode storageMode(bool deviceHasUnifiedMemory, WGPUBufferUsageFl
         return Buffer::createInvalid(*this);
 
     buffer.label = fromAPI(descriptor.label);
+    ASSERT(buffer.contents);
```

Comment:
storage mode can be private, isn't this wrong?

Reply:
Oh good point, I’ll remove these assertions